testbook/testbook.py

```diff
@@ -4,14 +4,22 @@
 
 
 class testbook:
-    def __init__(self, nb_path, prerun=None):
+    def __init__(self, nb_path, prerun=None, execute=False, timeout=60):
```

Comment:
What if we combine `prerun` and `execute`? If the argument is `True` we'll execute everything. If it's one of the normal prerun arguments then we do the prerun actions.

Reply:
Sure, makes sense. 

Also, do you think it would be better to have the default as - execute the entire notebook? And if we pass in `pre_run`, then it only executes those cells. 

So that way it would just be..

```python
@testbook('notebook.ipynb') # executes entire notebook
def test_foo(tb):
     # assertions
     pass
```